tools/clang/unittests/HLSL/ValidationTest.cpp

```diff
@@ -4247,24 +4247,24 @@ TEST_F(ValidationTest, UnitTestExtValidationSupport) {
   SetEnvVarW(L"DXC_DXIL_DLL_PATH", L"");
 
   // empty initialization should succeed
-  VERIFY_SUCCEEDED(ExtSupportEmpty.Initialize());
+  VERIFY_SUCCEEDED(ExtSupportEmpty.initialize());
 
-  VERIFY_IS_FALSE(ExtSupportEmpty.DxilDllFailedToLoad());
-  std::string EmptyPath = ExtSupportBogus.GetDxilDllPath();
+  VERIFY_IS_FALSE(ExtSupportEmpty.dxilDllFailedToLoad());
+  std::string EmptyPath = ExtSupportBogus.getDxilDllPath();
```

Comment:
Don't you mean to be querying the path from `ExtSupportEmpty` instead?
```suggestion
  std::string EmptyPath = ExtSupportEmpty.getDxilDllPath();
```